.github/workflows/ci.yml

```diff
@@ -6,6 +6,10 @@ on:
     - master
   schedule:
   - cron: '00 01 * * *'
+
+permissions:
+  contents: read #  to fetch code (actions/checkout)
```

Comment:
Could you add a comment explaining why this is here, and also when one might want to change or update it? Please also link to the GitHub docs page from your PR comment.

Reply:
Please take a look.